packages/flame_test/lib/src/test_golden.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flame/game.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:meta/meta.dart';
+
+/// Test that a game renders correctly.
+///
+/// The way golden tests work is as follows: you set up a scene in [testBody],
+/// then the test framework renders your game widget into an image, and compares
+/// that image against stored [goldenFile]. The test passes if two images are
+/// identical, or fails if the images differ even in a single pixel.
+///
+/// The term _golden file_ refers to the true rendering of a given game scene,
+/// captured at the creation of the test. In order to create a golden file, you
+/// first specify its desired name in the [goldenFile] parameter, and then run
+/// the tests using the command
+/// ```
+/// flutter test --update-goldens
```

Comment:
Would be nice to add this to melos too, so that you can run `melos update-goldens` from anywhere.

Reply:
Agreed!

Reply:
added